testimonials/block_womeng.md

```diff
@@ -0,0 +1,18 @@
+# Testimonials from Block's WomEng
+
+> I'm part of a smaller team, so I don't always get to interact with other women engineers on a daily basis. Even so, being involved with Block WomEng means that I never feel alone. Whether we're meeting over lunch to talk about the challenges we face, emailing links to empowering news articles, or chatting on our Slack channel, I feel like I'm a part of an engaged and thriving community of women in engineering (and our allies!). WomEng has helped me to find my voice as a software engineer, fight imposter syndrome, and become more confident in what I bring to the table. Even better, Square WomEng is helping me find ways that I can be a better ally to other underrepresented groups in tech. I'm so glad that I'm starting my career in tech here, as part of this amazing community.
+
+— Marie Chatfield ([@mariechatfield](https://twitter.com/mariechatfield))  
+_Former Software Engineer at Square_
```

Comment:
Should these be Block now too?

Reply:
```suggestion
_Former Software Engineer at Block_
```

Reply:
I left these as Square because all of these engineers left the company while it was still named Square. I think if we want to update it, it should be their business unit - Square, Cash App, TIDAL, etc. - based on direction from leadership on how to present ourselves to the public. For example, ahh sent an email that said we should set our LinkedIn to say we work at Square because we work on the Square product. The only people who would have "at Block" here are foundational. I'm not sure how to find out what product these folks worked on.